dbt/adapters/spark/impl.py

```diff
@@ -139,13 +144,54 @@ def add_schema_to_cache(self, schema) -> str:
     def _get_relation_information(self, row: agate.Row) -> RelationInfo:
         """relation info was fetched with SHOW TABLES EXTENDED"""
         try:
-            _schema, name, _, information = row
+            table_properties = {}
```

Comment:
I've changed this by directly separating the columns and properties when we parse the output. I think that makes more sense than splitting this later on because the `SHOW TABLES EXTENDED` output is different from the `DESCRIBE TABLE EXTENDED` output.